backend/src/repositories/job.repository.ts

```diff
@@ -377,42 +401,39 @@ export class JobRepository extends BaseRepository<typeof jobsDetails> {
             | "interviewing"
             | "rejected"
             | "hired"
-            | "withdrawn",
-        ),
+            | "withdrawn"
+        )
       );
     }
 
     const where = and(
-      ...(whereConditions.filter((c) => c !== undefined) as SQL<unknown>[]),
+      ...(whereConditions.filter((c) => c !== undefined) as SQL<unknown>[])
     );
 
     const items = await withDbErrorHandling(
       async () =>
         await db
           .select({
             application: jobApplications,
-            job: {
-              id: jobsDetails.id,
-              title: jobsDetails.title,
-              city: jobsDetails.city,
-              state: jobsDetails.state,
-              country: jobsDetails.country,
-              zipcode: jobsDetails.zipcode,
-              isRemote: jobsDetails.isRemote,
-              jobType: jobsDetails.jobType,
-            },
+            job: jobsDetails,
```

Comment:
I noticed you're returning the entire job object here vs returning some fields from the job table. I think it's better to return some fields so that we manage the amount of data we're sending over the network and the user already know details about the job since they applied for it. Did you have other reasons why you opted to send the entire object?